utils/difference.js

```diff
@@ -96,4 +110,9 @@ function nullDifferencePercents(row) {
 function allExist(...vals) {
   return vals.every(val => val !== undefined && val !== null);
 }
+
+function validateInputs(row, columns = [], callback) {
```

Comment:
is this supposed to be left here?

Reply:
Just removed it! :)

Reply:
@godfreyyeung let me know if you have more comments plz